src/AlgebraicGeometry/Schemes/SpecOpen/Morphisms/Constructors.jl

```diff
@@ -65,6 +66,7 @@ function restrict(
     issubset(U, domain(f)) || error("the given open is not an open subset of the domain of the map")
     issubset(V, codomain(f)) || error("the given open is not an open subset of the codomain of the map")
     issubset(preimage(f,V), U) || error("f(U) is not contained in V")
+    true
```

Comment:
```suggestion
```
These are SO annoying!!! Why do we keep on picking them up all the time???

Reply:
Sorry, I thought they were necessary. So I included them. I can remove them.

Reply:
Oh! Sorry, I didn't know that. I used to get these `true` statements from faulty merges and had to remove them again and again all over the place. My previous comment was not intended against your work!